moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -258,6 +258,12 @@ class Ros2ControlManager : public moveit_controller_manager::MoveItControllerMan
       if (!node_->has_parameter("ros_control_namespace"))
         ns_ = node_->declare_parameter<std::string>("ros_control_namespace", "/");
```

Comment:
I like your change   :+1:

It seems like this should be added too, for completeness:

```suggestion
      if (!node_->has_parameter("ros_control_namespace"))
        ns_ = node_->declare_parameter<std::string>("ros_control_namespace", "/");
      else
        node_->get_parameter<std::string>("ros_control_namespace", ns_);
```